charts/nginx-ingress-services/templates/certificate_federator.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if and .Values.tls.enabled .Values.tls.useCertManager .Values.federator.enabled }}
```

Comment:
I think should add a check which disallows disabling TLS when federator is enabled.

Reply:
What's the proper way to do this?

Reply:
We can use the `fail` keyword to make a failure in rendering the template. So something like:

```
{{- if and .Values.federator.enabled (not .Values.tls.enabled) }}
{{- fail "When federator is enabled, TLS must also be enabled" }}
{{- end }}
```